src/utils/utils.ts

```diff
@@ -122,3 +123,56 @@ export function isNode() {
     typeof process.versions.node !== 'undefined'
   );
 }
+
+/**
+ * Convert a base64 string to a Uint8Array for Node.js and browser environments.
+ * @returns A Uint8Array
+ */
+export function base64ToBytes(base64String: string): Uint8Array {
+  if (isNode()) {
+    return new Uint8Array(Buffer.from(base64String, 'base64'));
+  }
+  /* eslint-env browser */
+  const binString = atob(base64String);
+  return Uint8Array.from(binString, (m) => m.codePointAt(0));
+}
+
+/**
+ * Decode a base64 string for Node.js and browser environments.
+ * @returns A decoded string
+ */
+export function base64ToString(base64String: string): string {
```

Comment:
I'm wondering if it would be more modular to offer a separate set bytes <=> string conversion functions. For instance, if I had a hex string I wanted to convert to a normal string, right now I'm not able to do that.

These might help us pinpoint the current issue we're debugging as well.

Reply:
Feel free to downgrade this to a nit -- I don't feel as strongly anymore, and we can easily add more util functions if we find they'd be useful.